### DISCUSSION TITLE
💡 Look into supporting forced colors

### DISCUSSION THREAD

**Discussion by fraincs** [gsoft-inc/sg-orbit#1113] [Ideas]
## Problem to solve

Forced colors are an accessibility feature that can be turned on by users, Orbit components are not currently reacting to this.
See: https://polypane.app/blog/forced-colors-explained-a-practical-guide

## Idea

Test that Orbit components behave well with this turned on.

**Comment by patricklafrance** [comment]
Is it really something Orbit want to support?

Does other DS support this?

I would rather add additional themes (like high constrat theme( that we control and fit our branding.

What do you think?

**Comment by fraincs** [comment]
This was an idea / discussion and not a task. These are good questions and I would totally do a high contrast theme before a forced color support, although forced colors requires no input from design and would be a win for accessibility. Is it common, I would have to check if some DS are supporting it by default.

**Comment by patricklafrance** [comment]
I would move this to a discussion and see if design is open to it. Forced colors are problematic from a branding issue and might even result in fact I guess to other accessibility issue like bad color contrast.

**Comment by fraincs** [comment]
Totally but the user would be aware of this. Which makes me think we indeed should move this to a discussion instead.
